### PR TITLE
Updated contracts to support editing mail categories

### PR DESCRIPTION
**Pull request by LukaszKorz**
Updated contracts to support editing mail categories and returning categories in Piece contract

### CODE REVIEW

**Comment by LukaszKorz** [comment]
[Show mail category in item overview](https://trello.com/c/n4YOFQq1/734-show-mail-category-in-item-overview)

File: Iris.Sdk.Contracts/Inbox/Piece.cs [hunk 1]
```diff
@@ -36,33 +36,36 @@ public Piece()
         [DataMember(Name = "attributes")]
         public List<string> Attributes;
 
+        //[DataMember(Name = "categories")]
```
Comment (MartyIce) [inline Iris.Sdk.Contracts/Inbox/Piece.cs:39]: If we are no longer using it, can you just delete?

**Comment by hadaszpatryk** [comment]
I've had no time to review the whole solution yet but this will change the API contract, right? just leaving a note here to remember about testing/considering existing API consumers (mobile, ZAP, etc)

**Comment by LukaszKorz** [comment]
> I've had no time to review the whole solution yet but this will change the API contract, right? just leaving a note here to remember about testing/considering existing API consumers (mobile, ZAP, etc)

@hadaszpatryk  The only change in existing contracts was updating ```public List<string> Categories;``` to ```public List<PieceCategory> Categories;``` in Piece contract. I've checked it with @mogrodniczak and this field isn't used anywhere

File: Iris.Sdk.Contracts/Inbox/Piece.cs (hunk 1 above)
Comment (LukaszKorz) [reply Iris.Sdk.Contracts/Inbox/Piece.cs]: Just forgot to remove it when updating contracts

**Pull request closed without merging**